File: JavaOOP/interface/multiple-1/Student.php
<?php
class Student {
	public $id;
	public $name;
	
	function setStudent($id, $name) {
		$this -> id=id;
		$this -> name=name;
	}
	
	function getStudent() {
		echo $this -> id;
		echo $this -> name;
	}
	
}

$Stu = new Student;
$Stu->setStudent(200, "Jenothan");
$Stu->getStudent();

?>

File: JavaOOP/method over loading/Student.java
class Student{
	int id;
	String name;
	
	void setStudent(){
		this.id= 1000;
		this.name= "Seelan";
	}
	
	void setStudent(int id){
		this.id= id;
		this.name= "Seelan";
	}
	
	void setStudent(String name){
		this.id= 1000;
		this.name= name;
	}
	
	void setStudent(int id, String name){
		this.id= id;
		this.name= name;
	}
	
	void setStudent(String name, int id){
		this.id= id;
		this.name= name;
	}
	
	void getStudent(){
		System.out.println("id "+this.id);
		System.out.println("name "+this.name);
	}
}